Exercise_1/problem-2.go

```diff
@@ -0,0 +1,67 @@
+package main
+
+import (
+	"encoding/csv"
+	"flag"
+	"fmt"
+	"io"
+	"os"
+	"time"
+)
+
+var set int = 0 //used to switch between go routines
+var count int = 0
+var total int = 0
+
+func main() {
+
+	var path string
+	fmt.Printf("Enter the path to the csv file:")
+	fmt.Scan(&path)
+	pathptr := flag.String("Path", path, "Path to the csv file")
+	flag.Parse()
```

Comment:
If the user doesn't provide a filename, the program should automatically default to problems.csv . Please add such functionality.